packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-views.test.ts.snap

```diff
@@ -0,0 +1,47 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`amplify view renderer tests should generate a table element 1`] = `
+"<Table>
+  {!disableHeaders && (
+    <TableHead>
+      <TableRow>
+        <TableCell as=\\"th\\">hireDate</TableCell>
+        <TableCell as=\\"th\\">comments</TableCell>
+        <TableCell as=\\"th\\">createdAt</TableCell>
+        <TableCell as=\\"th\\">updatedAt</TableCell>
+      </TableRow>
+    </TableHead>
+  )}
+  <TableBody>
+    {items.map((item, index) => (
+      <TableRow
+        onClick={
+          highlightOnHover && onRowClick ? () => onRowClick(item, index) : null
```

Comment:
fyi, I'm not 100% the highlightOnHover needs to be enabled for onRowClick, we may need to change this later.